autoload/ale/floating_preview.vim

```diff
@@ -47,16 +47,53 @@ function! ale#floating_preview#Show(lines, ...) abort
         endif
     augroup END
 
-    let l:width = max(map(copy(a:lines), 'strdisplaywidth(v:val)'))
-    let l:height = min([len(a:lines), 10])
+    let [l:lines, l:width, l:height] = s:PrepareWindowContent(a:lines)
+
     call nvim_win_set_width(w:preview['id'], l:width)
     call nvim_win_set_height(w:preview['id'], l:height)
-
-    call nvim_buf_set_lines(w:preview['buffer'], 0, -1, v:false, a:lines)
+    call nvim_buf_set_lines(w:preview['buffer'], 0, -1, v:false, l:lines)
     call nvim_buf_set_option(w:preview['buffer'], 'modified', v:false)
     call nvim_buf_set_option(w:preview['buffer'], 'modifiable', v:false)
 endfunction
 
+function! s:PrepareWindowContent(lines) abort
+    let l:max_height = 10
```

Comment:
Any reason why the max height is not a global value? I want to set it in my init.vim

Reply:
I follow the original setting. If you need, you could send a pull request.